libzcashlc.podspec

```diff
@@ -0,0 +1,15 @@
+Pod::Spec.new do |s|
+    s.name             = 'libzcashlc'
+    s.version          = '0.0.1'
+    s.summary          = 'Rust core for ZCash clients'
+    s.homepage         = 'https://github.com/dh-ecc/libzcashlc'
+    s.license          = { :type => 'MIT', :file => 'LICENSE' }
+    s.author           = { 
+        'Francisco Gindre' => 'francisco.gindre@gmail.com',
+        'Jack Grigg' => 'str4d@electriccoin.co'
+     }
+    s.source           = { :git => 'https://github.com/dh-ecc/libzcashlc', :tag => s.version.to_s }
```

Comment:
shouldn't this have the url for this repo?

Reply:
great spot! 

Yes it should! I managed to revert this somehow :(

Reply:
👍 done in [39ffd1a](https://github.com/zcash-hackworks/zcash-light-client-ffi/pull/1/commits/39ffd1a7ca87e958e7911e1c3aec5bc90c78c664)